phidgets_current_inputs/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>phidgets_current_inputs</name>
+  <version>1.0.5</version>
+  <description>Driver for the Phidgets Current Input devices</description>
+
+  <maintainer email="martin.guenther@dfki.de">Martin Günther</maintainer>
+  <maintainer email="clalancette@openrobotics.org">Chris Lalancette</maintainer>
+
+  <license>BSD</license>
+
+  <author>Chris Lalancette</author>
```

Comment:
By "I wrote essentially no code", you mean that you simply followed the pattern of the other phidgets and did some text replacement? That still counts as writing code, and the author should be you. It's better to leave the maintainer fields as is, because it's @clalancette and me who will be maintaining the code once it's merged. I think the maintainer field is parsed by the build farm to send out emails on build failures, and Chris and me want to get those. :)